public/app/app.ts

```diff
@@ -126,6 +129,11 @@ if (process.env.NODE_ENV === 'development') {
 
 export class GrafanaApp {
   context!: GrafanaContextType;
+  extensionRegistries!: {
+    addedLinks: AddedLinksRegistry;
+    addedComponents: AddedComponentsRegistry;
+    exposedComponents: ExposedComponentsRegistry;
+  };
```

Comment:
Could use the  in `PluginExtensionRegistries` type in `/public/app/features/plugins/extensions/registry/types`

Reply:
Then you can replace line 223 with `this.pluginExtensionsRegistries = setupPluginExtensionRegistries();`

Reply:
That's a good point, will do 👍